### PR TITLE
Fix/1435 settings in full screen mode

### PR DESCRIPTION
**Pull request by navdeepsingh**
@plug-n-play please review.
Staging Link: https://sttm-dev-fix-1435-setti-hhkooz.herokuapp.com/

* [x]  Passed [sanity tests](http://bit.ly/sttm-sanity-tests).
* [x]  Ran `npm test` & fixed newly introduced lint errors.
* [x]  Checked console for errors.

<!-- New to markdown? Simply put an 'x' between [ ] to check it. Like [x] this. (No spaces).

### CODE REVIEW

File: src/js/components/ShareButtons.js [hunk 1]
```diff
@@ -199,9 +199,21 @@ class ShareButtons extends React.PureComponent {
...
 
     return (
-      <div id="share-menu">
-        <ul className="share-buttons">{media.map(item => mediaMap[item])}</ul>
-      </div>
+      <>
+        <div id="share-menu">
+          <ul className="share-buttons">{media.map(item => mediaMap[item])}</ul>
+        </div>
+        <div id="fullscreen-share-menu">
```
Comment (plug-n-play) [inline src/js/components/ShareButtons.js:206]: Why has this been made part of ShareButtons component? It doesn't seem to belong here.

File: src/scss/_sharing.scss [hunk 2]
```diff
@@ -36,6 +36,17 @@
   }
 }
 
+#fullscreen-share-menu {
```
Comment (plug-n-play) [inline src/scss/_sharing.scss:39]: Same for other things. This is not the right place to add this component and related styling.

**Review by plug-n-play — CHANGES_REQUESTED** [review]
Either use an existing appropriate component for this change. If it doesn't exists, please create a new one.

**Comment by saintsoldierx** [comment]
@plug-n-play please review today if you can. Thanks

**Review by plug-n-play — APPROVED** [review]

**Comment by plug-n-play** [comment]
Thank you @navdeepsingh

Good job!

**Comment by navdeepsingh** [comment]
> Thank you @navdeepsingh
> 
> Good job!

Thanks @plug-n-play for the compliment. It matters to me a lot.

**Pull request merged**